iset.mm

```diff
@@ -155556,6 +155607,53 @@ WLPO is known to not be provable in IZF (and most constructive
       VLVMLNVNVRWEWFVJVOLNVPVLVS $.
   $}
 
+  ${
+    $d x y $.
+    $( Real trichotomy implies decidability of real number equality.  Or in
+       other words, analytic LPO implies analytic WLPO (see ~ trilpo and
+       ~ redcwlpo ).  Thus, this is an analytic analogue to ~ lpowlpo .
+       (Contributed by Jim Kingdon, 24-Jul-2024.) $)
+    tridceq $p |- ( A. x e. RR A. y e. RR ( x < y \/ x = y \/ y < x )
+        -> A. x e. RR A. y e. RR DECID x = y ) $=
+      ( cv clt wbr weq w3o cr wral wdc wcel wa wi ltne ex adantr wo syl6 sylibr
+      wne olc necom dcne 3imtr4i orc a1i adantl 3jaod ralimdva ralimia ) ACZBCZ
+      DEZABFZULUKDEZGZBHIUNJZBHIAHUKHKZUPUQBHURULHKZLZUMUQUNUOUTUMULUKTZUQURUMV
+      AMUSURUMVAUKULNOPUKULTZUNVBQZVAUQVBUNUAZULUKUBUKULUCZUDRUNUQMUTUNVCUQUNVB
+      UEVESUFUTUOVBUQUSUOVBMURUSUOVBULUKNOUGVBVCUQVDVESRUHUIUJ $.
+  $}
+
+  ${
+    $d x y z $.
+    $( Two ways to express decidability of real number equality.  (Contributed
+       by Jim Kingdon, 23-Jul-2024.) $)
+    redc0 $p |- ( A. x e. RR A. y e. RR DECID x = y <->
+        A. z e. RR DECID z = 0 ) $=
```

Comment:
I guess this statement confirms the above possibility.

Reply:
Yes, although `redc0` is for analytic WLPO. `reap0` is the one for analytic LPO which is what you had asked about.